### ISSUE TITLE
[roadshow] creating nationalparks-live via webui results in no route

### ISSUE THREAD

**Issue by thoraxe** [openshift-evangelists/openshift-workshops#37]
deploying from the image doesn't create a route. It needs to be created manually.

**Comment by siamaksade** [comment]
The exercise includes creating a route through the web ui.

**Comment by thoraxe** [comment]
Yes looks like I missed this. Which seems like a problem. I'll add a picture for it and tweak the description.
